src/search.js

```diff
@@ -1,31 +1,31 @@
-import {assoc, concat, filter, has, isEmpty, map, pipe, prop, sortBy} from "ramda"
+import {ascend, assoc, concat, descend, has, isEmpty, map, prop, sortBy, sortWith} from "ramda"
 import * as diacritics from 'diacritics'
 
 
 export function findParametersAndVariables(parameters, variables, query) {
-  function matchesQuery(parameterOrVariable) {
-    if (isEmpty(normalizedQuery)) {
-      return true
-    }
-    return parameterOrVariable.name.includes(normalizedQuery) ||
-      parameterOrVariable.searchIndex.includes(normalizedQuery)
+  function weightByPositionIn(string, substring) {
```

Comment:
> Do you really need to specify in the function name _how_ the weight is computed, when there currently is only one way to weigh strings? Maybe `weightIn` is enough here? :)
> Also, this is not exactly the _weight_ in the usual sense, since the higher it is the lower the importance :wink: “inferiority”, maybe? 🤔